src/presets/ahooks.ts

```diff
@@ -0,0 +1,31 @@
+import { readFileSync } from 'fs'
+import { resolveModule } from 'local-pkg'
+import type { PackageIndexes } from '@vueuse/metadata'
+import type { ImportsMap } from '../types'
+
+let _cache: ImportsMap | undefined
+
+export default (): ImportsMap => {
+  if (!_cache) {
+    let indexesJson: PackageIndexes | undefined
+    try {
+      const corePath = resolveModule('ahooks') || process.cwd()
+      const path = resolveModule('ahooks/metadata.json')
+          || resolveModule('ahooks/metadata.json', { paths: [corePath] })
```

Comment:
```suggestion
      const path = resolveModule('ahooks/metadata.json')
```

In VueUse we did that because we have changed the format in v8 and the code is for backward compatibility. In ahooks there is no such issue and we could keep it simple.

Reply:
Furthermore, should we provide a new feature that any libraries export the standard `metadata.json` can use this plugin directly instead of adding one more preset~~